src/lib_cxx/data/parser/cxx/GeneratePCHAction.cpp

```diff
@@ -36,14 +37,11 @@ std::unique_ptr<clang::ASTConsumer> GeneratePCHAction::CreateASTConsumer(
 	const auto& FrontendOpts = CI.getFrontendOpts();
 	auto Buffer = std::make_shared<clang::PCHBuffer>();
 	std::vector<std::unique_ptr<clang::ASTConsumer>> Consumers;
-	Consumers.push_back(llvm::make_unique<clang::PCHGenerator>(
-		CI.getPreprocessor(),
-		OutputFile,
-		Sysroot,
-		Buffer,
+	Consumers.push_back(std::make_unique<clang::PCHGenerator>(
+		CI.getPreprocessor(), CI.getModuleCache(), OutputFile, Sysroot, Buffer,
 		FrontendOpts.ModuleFileExtensions,
-		true,
-		FrontendOpts.IncludeTimestamps));
+		CI.getPreprocessorOpts().AllowPCHWithCompilerErrors,
```

Comment:
Oh, just a copy-paste from the clang code, should this still be "true" here?

Reply:
yes

Reply:
changed